### PR TITLE
feat: Allows skipping artefacts upload 4 GH Release

### PR DESCRIPTION
**Pull request by ElvenSpellmaker**
# Pull Request Template

## 📲 What

feat: Allows skipping artefacts upload for a GitHub Release

## 🤔 Why

Not all projects have artefacts to upload

## 🕵️ How to test

### CODE REVIEW

File: src/modules/EnsonoBuild/exported/Publish-GitHubRelease.ps1 [hunk 1]
```diff
@@ -87,11 +92,18 @@ function Publish-GitHubRelease() {
...
+    # If the artifactsList is empty, get all the files in the specified
+    # artifactsDir otherwise find the files that have been specified
     if ($artifactsList.Count -eq 0) {
-        $artifactsList = Get-ChildItem -Path $artifactsDir -Recurse -File
+        if ($uploadArtifacts) {
+            $artifactsList = Get-ChildItem -Path $artifactsDir -Recurse -File
+        }
     } else {
+        if (!$uploadArtifacts) {
+            Write-Error "'uploadArtifacts' is set to false, but 'artifactsList' isn't empty..."
```
Comment (dnitsch) [inline src/modules/EnsonoBuild/exported/Publish-GitHubRelease.ps1:103]: maybe this is better as a warning...

**Review by dnitsch — APPROVED** [review]

File: src/modules/EnsonoBuild/exported/Publish-GitHubRelease.ps1 (hunk 1 above)
Comment (ElvenSpellmaker) [reply src/modules/EnsonoBuild/exported/Publish-GitHubRelease.ps1:103]: Yeah, I'll think on it, but I feel like if the user is supplying a list and they don't want to upload something has gone wrong

**Pull request merged**